src/profanity.ts

```diff
@@ -20,6 +20,7 @@ export class Profanity {
   }
 
   exists(text: string): boolean {
+    text = typeof text === 'string' ? text : '';
```

Comment:
This works, but let's use a guard clause instead so we don't process empty strings once we know the input is an invalid type:

```js
exists(text: string): boolean {
 if (typeof text !== 'string') {
   return false;
 }
```

I noticed the same issues occurs with `censor`. Will you please add a similar guard clause to `censor` while you're at it?

```js
censor(text: string): string {
 if (typeof text !== 'string') {
   return text;
 }
```

Reply:
Sure thing. I have changed/added guards to exists and censor.